src/Components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Tabs.scss';
+
+type Props = {
+  tabChangeMethod: (tabId: Tab) => void,
+  tabs: Tab[],
+  selectedTabId: string,
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabChangeMethod,
+  tabs,
+  selectedTabId,
+}) => {
+  const selectedObj = tabs.find(tab => tab.id === selectedTabId);
+
+  const classNameMetod = (currentTabId: string) => {
+    if (currentTabId === selectedTabId) {
+      return 'tabs__button active';
+    }
+
+    return 'tabs__button';
+  };
+
+  return (
+    <div className="tabs">
+      <div>
```

Comment:
It is not needed div

Reply:
Disagree with you. It"s nessesary for classname